apps/estuary/test/unit/estuary/query/select_test.exs

```diff
@@ -2,126 +2,145 @@ defmodule Estuary.Query.SelectTest do
-        "table_name" => "any_table",
-        "limit" => 1000
-      }
-
-      allow(PrestigeHelper.create_session(), return: :do_not_care)
-      allow(Prestige.query!(any(), any()), return: :do_not_care)
-      allow(Prestige.Result.as_maps(any()), return: @expected_events)
-      assert {:ok, @expected_events} == Select.select_table(table_schema)
-    end
-
-    @tag capture_log: true
-    test "should return in ascending order when order is missing" do
-      expected_events = [
-        %{
-          "author" => "Author-2020-01-21 23:25:52.522084Z",
-          "create_ts" => 1_579_649_152,
-          "data" => "Data-2020-01-21 23:25:52.522107Z",
-          "type" => "Type-2020-01-21 23:25:52.522111Z"
-        },
-        %{
-          "author" => "Author-2020-01-21 23:29:20.171519Z",
-          "create_ts" => 1_579_649_360,
-          "data" => "Data-2020-01-21 23:29:20.171538Z",
-          "type" => "Type-2020-01-21 23:29:20.171543Z"
-        }
-      ]
-
-      table_schema = %{
-        "columns" => ["author", "create_ts", "data", "type"],
-        "table_name" => "any_table",
-        "order_by" => "create_ts",
-        "limit" => 1000
-      }
-
-      allow(PrestigeHelper.create_session(), return: :do_not_care)
-      allow(Prestige.query!(any(), any()), return: :do_not_care)
-      allow(Prestige.Result.as_maps(any()), return: expected_events)
-      assert {:ok, expected_events} == Select.select_table(table_schema)
-    end
-
-    @tag capture_log: true
-    test "should return error when table name is missing" do
-      expected_error = {:error, %RuntimeError{message: "Table name missing"}}
-
-      table_schema = %{
-        "columns" => ["author", "create_ts", "data", "type"],
-        "order_by" => "create_ts",
-        "order" => "DESC",
-        "limit" => 1000
-      }
-
-      assert expected_error == Select.select_table(table_schema)
-    end
+
+  @tag capture_log: true
+  test "should create the query with * and the table name without any condition with limit all, when only table name is passed no column and no limit is passed" do
+    expected_select_statement = "SELECT *\n      FROM any_table\n      \n      \n      LIMIT ALL"
+
+    table_schema = %{
+      "table_name" => "any_table"
+    }
+
+    actual_select_statement = Select.create_select_statement(table_schema)
+    assert expected_select_statement == actual_select_statement
+  end
+
+  @tag capture_log: true
+  test "should return error, when table name is missing" do
+    expected_error = %RuntimeError{message: "Table name missing"}
+    table_schema = %{}
+    actual_error = Select.create_select_statement(table_schema)
+    assert expected_error == actual_error
+  end
+
+  @tag capture_log: true
+  test "should create the query with * with table name without any condition with limit 1000, when limit is passed" do
+    expected_select_statement = "SELECT *\n      FROM any_table\n      \n      \n      LIMIT 1000"
```

Comment:
I don't think we should expect line breaks and tabs in these generated statements.